src/icons/icons.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import { storiesOf } from '@storybook/react';
-import path from 'path';
-import React from 'react';
+import * as path from 'path';
```

Comment:
is this correct?

Reply:
it seem to work but I will remove it it is by accident